double-press-double-switch.js

```diff
@@ -0,0 +1,79 @@
+// the script show how perform mutliple action with a double classic switch
+// it support simple press and double press on each switch and press on both button at the same time 
+
+let timer = undefined;
+let previouHitButtonId = undefined;
+
+let simpleClickAction1 = 'http://shelly1-ip/rpc/Switch.Toggle?id=0';
+let doubleClickAction1 = 'http://shelly1-ip/rpc/Switch.Toggle?id=1';
+let simpleClickAction2 = 'http://shelly2-ip/rpc/Switch.Toggle?id=0';
+let doubleClickAction2 = 'http://shelly2-ip/rpc/Switch.Toggle?id=1';
+
+let simpleClickAction1FullOff = 'http://shelly1-ip/rpc/Switch.Set?id=0&on=false';
+let doubleClickAction1FullOff = 'http://shelly1-ip/rpc/Switch.Set?id=1&on=false';
+let simpleClickAction2FullOff = 'http://shelly2-ip/rpc/Switch.Set?id=0&on=false';
+let doubleClickAction2FullOff = 'http://shelly2-ip/rpc/Switch.Set?id=1&on=false';
+
+let doubleClickDelay = 250;
```

Comment:
The standard in our scripts is to have a CONFIG object where you define all the script configs.
It allows for users to locate the place where they can change device configuration

Reply:
I added CONFIG object for the two scripts